R/createInputForDescriptives.r

```diff
@@ -174,6 +178,10 @@ createInputForDescriptives.GADSdat <- function ( GADSdat, idExpr = "^ID", impExp
                               if (verbose) {cat(paste0("Scale '",sc,"' (not imputed): Found following ",length(items)," items: '",paste(items, collapse="', '"),"'.\n"))}
                               v[eatTools::whereAre(c(items, sc), v[,"varName"], verbose=FALSE),"group"] <- sc
                         }
+                        if ("fake_item" %in% v[,"type"]) {
+                              v[grep("fake_item",v[,"type"]) ,"group"] <- eatTools::removeNumeric(v[grep("fake_item",v[,"type"]) ,"varName"])
+                              warning("Identification of fake scales cannot be done completely automatically. Please check if the assignment of which items belong to a common scale is correct.")

```

Comment:
LGTM. 

Nit: I think I would prefer not to issue a warning. IMHO all guesses by `createInputForDescriptives()` can only be very approximate, and this applies to imputed variables and scales, as well. Issueing such a warning would suggest that users do not have to worry about anything else then fake-scales.

Reply:
ok, should I changed it to a message instead? or remove it completely? (I would rather not remove it completely, as users should be made aware here that the automatic mapping might be inaccurate, and they would need to double check it.)